packages/web/src/components/result/addons/Results.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { getClassName, isEqual } from '@appbaseio/reactivecore/lib/utils/helper';
+import types from '@appbaseio/reactivecore/lib/utils/types';
+
+class Results extends React.Component {
+	shouldComponentUpdate(nextProps) {
+		/*
+			Use shouldComponentUpdate because PureComponent uses shallow comparison of objects.
+			If props contain complex data structures, it may produce false-negatives for
+			deeper differences.
+		*/
+		return !isEqual(nextProps, this.props);
```

Comment:
Would it not create performance issues, suppose you're rendering 50+ results in a page.
It'll be slow to compare props (results) with nested values.
I think PureComponent will be enough.

Reply:
I have added the comment for the same do check it. 

I used `PureComponent` component before but it was re-rendering the result.

Reply:
Here is the screen of Profiler and console in both scenario for 100 Results:

#### With `shouldComponentUpdate`
![image](https://user-images.githubusercontent.com/22376783/63488622-4535bc00-c4cd-11e9-8f31-bdbf0d33bc98.png)
![image](https://user-images.githubusercontent.com/22376783/63488843-ffc5be80-c4cd-11e9-9c74-5f5266a5ed58.png)


#### With `PureComponent`
![image](https://user-images.githubusercontent.com/22376783/63488682-7b733b80-c4cd-11e9-8c70-e3386879087a.png)
![image](https://user-images.githubusercontent.com/22376783/63488727-a2ca0880-c4cd-11e9-91ba-cfb13e34ba1d.png)